app/assets/stylesheets/markdown-editor.scss

```diff
@@ -27,6 +27,7 @@
 
   .btn-group {
     margin-bottom: 8px;
+    &:first-child { margin-left: 0; }
```

Comment:
This line was to move the first group to the beginning (same margin as from the top/bottom):

![](https://user-images.githubusercontent.com/458548/29236691-819003b0-7f0e-11e7-9696-373111987a8d.png)

Without this line:

![](https://user-images.githubusercontent.com/458548/29236703-d02d03f6-7f0e-11e7-8e18-7284b616c8e9.png)